gtsfm/data_association/point3d_initializer.py

```diff
@@ -67,8 +69,9 @@ class TriangulationOptions(NamedTuple):
     See the following slides for a derivation of the #req'd samples: http://www.cse.psu.edu/~rtc12/CSE486/lecture15.pdf
 
     Args:
-        reproj_error_threshold: the maximum reprojection error allowed.
         mode: triangulation mode, which dictates whether or not to use robust estimation.
+        reproj_error_threshold: the maximum reprojection error allowed.
+        min_triangulation_angle: the minimum angle (in degrees) subtended at the triangulated point from 2 cameras.
```

Comment:
mention that this is a threshold, and that tracks for which the max angle at any two cameras < threshold will be rejected.